src/components/auth/Registration.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Button,
+  Link,
+  TextField,
+} from '@mui/material';
+import Header from '../../elements/Header';
+import Logo from '../../elements/Logo';
+import './Auth.scss';
+
+const Registration = () => {
+  const [login, setLogin] = useState('');
+  const [password, setPassword] = useState('');
+  const [retryPassword, setRetryPassword] = useState('');
```

Comment:
why do you need so many states?

Reply:
i rebuild it. input value `login`, `password` and `retryPassword` have one state. 
i make not simple error alert functional. i don't can use one state for errors. it's add more code.